build.gradle

```diff
@@ -1,52 +1,56 @@
 plugins {
     id 'java-library'
-    id 'io.freefair.lombok' version '5.1.0'
     id 'maven-publish'
     id 'signing'
     id 'io.github.gradle-nexus.publish-plugin' version '1.1.0'
-    id 'net.nemerosa.versioning' version '2.8.2'
+    id 'net.nemerosa.versioning' version '2.15.1'
 }
 
 group = 'com.github.instagram4j'
 archivesBaseName = 'instagram4j'
 // For release
 // Must be on release/<version> branch
 // Release: ./gradlew publishToSonatype closeSonatypeStagingRepository --console=plain
-version = "${versioning.info.branchType.equals('release') ? versioning.info.display : versioning.info.full}"
+version = "${versioning.info.branchType == 'release' ? versioning.info.display : versioning.info.full}" as Object
 
 java {
-    sourceCompatibility = JavaVersion.VERSION_1_8
-    targetCompatibility = JavaVersion.VERSION_1_8
+    sourceCompatibility = JavaVersion.VERSION_17
+    targetCompatibility = JavaVersion.VERSION_17
 }
 
 repositories {
-    jcenter()
+    mavenCentral()
 }
 
 dependencies {
     // OkHttp
-    api 'com.squareup.okhttp3:okhttp:4.7.2'
-    api 'com.squareup.okhttp3:okhttp-urlconnection:4.7.2'
+    api 'com.squareup.okhttp3:okhttp:4.10.0'
+    api 'com.squareup.okhttp3:okhttp-urlconnection:4.10.0'
     // jackson data-bind
-    api 'com.fasterxml.jackson.core:jackson-databind:2.11.0'
-    api 'com.fasterxml.jackson.core:jackson-annotations:2.11.0'
-    api 'org.slf4j:slf4j-api:1.7.30'
+    api 'com.fasterxml.jackson.core:jackson-databind:2.14.1'
+    api 'com.fasterxml.jackson.core:jackson-annotations:2.14.1'
+    api 'org.slf4j:slf4j-api:2.0.6'
     // Test utilities
-    testImplementation 'org.slf4j:slf4j-log4j12:1.7.30'
-    testImplementation 'com.squareup.okhttp3:logging-interceptor:4.7.2'
+    testImplementation 'org.slf4j:slf4j-reload4j:2.0.6'
+    testImplementation 'com.squareup.okhttp3:logging-interceptor:4.10.0'
     testImplementation 'log4j:log4j:1.2.17'
-    testImplementation 'junit:junit:4.12'
+    testImplementation 'junit:junit:4.13.2'
     testImplementation 'pl.pragmatists:JUnitParams:1.1.1'
+
+    compileOnly 'org.projectlombok:lombok:1.18.22'
```

Comment:
NACK, why add more dependencies, just update old ones and keep the commit in scope.

Reply:
It's actually to remove the 'io.freefair.lombok' gradle plugin, so dependencies are not added (the plugin actually does this adjustments automatically)